src/OpenTelemetry.Contrib.Instrumentation.EventCounters/Implementation/MeterTelemetryPublisher.cs

```diff
@@ -57,19 +56,16 @@ private void StoreValue(MetricKey metricKey, MetricInstrument instrument, Metric
         {
             switch (instrument.Type)
             {
-                case MetricType.LongSum:
-                case MetricType.LongGauge:
-                    this.longValueStore[metricKey] = (long)metricTelemetry.Sum;
+                case InstrumentationType.Counter:
+                case InstrumentationType.Gauge:
+                    this.longValueStore[metricKey] = (long)metricTelemetry.Value;
                     break;
-                case MetricType.DoubleGauge:
-                case MetricType.DoubleSum:
-                    this.doubleValueStore[metricKey] = metricTelemetry.Sum;
-                    break;
-                case MetricType.Histogram:
```

Comment:
How did you make sure this is not needed?

Reply:
cijo mentioned that histogram is not needed